aria-practices.html

```diff
@@ -4586,7 +4586,7 @@ <h4><span id="naming_role_guidance_heading">Accessible Name Guidance by Role</sp
             </tr>
             <tr>
               <td><a href="#marquee" class="role-reference"><code>marquee</code></a></td>
-              <td>Required</td>
+              <td>Discretionary</td>
               <td>Use <code>aria-labelledby</code> if a visible label is present, otherwise use <code>aria-label</code>.</td>
```

Comment:
I think this is ok but we could consider changing this to match the other discretionary roles which describe why you might want to give it an accessible name. However - I'm not sure I can really come up with a valid use case for that.

Reply:
It's valid to give a marquee a name, just didn't make sense to require it.

I very quickly tested [4 marquees](https://carmacleod.github.io/playground/marquee-test.html) using jaws/nvda in chrome/ff.

Notes:
- The first 2 marquees use (obsolete) HTML `<marquee>` elements. One has `aria-labelledby` and the other has `aria-label`.
- The next 2 are fake marquees that use a span (I didn't add any animation). As above, one has `aria-labelledby` and the other has `aria-label`.
- Both browsers expose the role correctly, but I had to use `role="marquee"` ("belts and braces" style) on the 2 HTML `<marquee>` elements (otherwise, they just had a generic text container role).
- Both browsers expose all 4 accessible names correctly, and they also expose the `aria-live="off"`.
- Both screen readers read the `aria-labelledby` span, because it's visible text content. :)
- Neither screen reader read the `aria-label`.
- Both screen readers read all of the marquee's content, i.e. they ignored the animated scrolling.

So we could say something along the lines of, "At this time, screen readers do not support marquee role well. When naming a marquee, `aria-labelledby` with a visible label is recommended, because some screen readers do not read an `aria-label` on a marquee."

I'll try Safari/VO tomorrow.